fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1566,6 +1553,16 @@ ACTOR Future<Void> tLog(
 						when(wait(waitForAll(tlogGroupRecoveries))) {}
 					}
 
+					// Disk errors need a chance to kill this actor.
+					wait(delay(0.000001));
+
+					for (auto& it : self->tlogGroups) {
+						Reference<TLogGroupData> tlogGroup = it.second;
```

Comment:
Nit: could save one line and be more readable like: 
```suggestion
					for(auto& [_, tlogGroup] : self->tlogGroups) {
```

Reply:
will fix.